continunet/tests/conftest.py

```diff
@@ -23,3 +23,54 @@ def fits_file(tmp_path):
 
     yield path
     path.unlink()
+
+
+@pytest.fixture
+def trained_model():
+    """Fixture for a trained model."""
+    return "continunet/network/trained_model.h5"
```

Comment:
Are you anticipating that these may later be computed? For constants like a string literal or something you can probably just use a global constant.

Personally I've only really used fixtures when I needed a test resource that has to be cleaned up (e.g., spawning a server, yielding the instance, and tearing down after the test, or some file system things).

If you're anticipating doing the same thing, then ignore this comment!

Reply:
You're not going to like this, but I like it this way then I don't have to import it 🫠